src/TestFramework/PhpUnit/CommandLine/ArgumentsAndOptionsBuilder.php

```diff
@@ -123,6 +141,34 @@ public function buildForMutant(string $configPath, string $extraOptions, array $
         return $options;
     }
 
+    private function mapSourceClassToTestClass(SplFileInfo $sourceFile): string
+    {
+        return sprintf('%sTest', $sourceFile->getBasename('.' . $sourceFile->getExtension()));
+    }
+
+    /**
+     * @return list<string>
+     */
+    private function prepareArgumentsAndOptions(string $configPath, string $extraOptions): array
```

Comment:
could be static

Reply:
it's not a common practice in Infection's codebase, so I decided to leave it as is. 

If we decide to use static methods every time there is no `$this` in it - let's discuss it and probably integrate a fixer? But I wonder what's the benefit and whether it worth doing it.